etc/derecho/build_tuvx_derecho_gnu.sh

```diff
@@ -23,29 +23,10 @@ if [[ ! -d "${TUVX_HOME}" ]]; then
   return
 fi
 
-echo "Building JSON Fortran"
-
-# get & build the source code of JSON Fortran
-
-cd ${TUVX_HOME}
-curl -LO https://github.com/jacobwilliams/json-fortran/archive/8.3.0.tar.gz
-tar -zxvf 8.3.0.tar.gz
-cd json-fortran-8.3.0
-mkdir build
-cd build
-INSTALL_DIR=$TUVX_HOME/json-fortran-8.3.0
-cmake -D SKIP_DOC_GEN:BOOL=TRUE -D CMAKE_INSTALL_PREFIX=$INSTALL_DIR ..
-make install
-
+# download and build TUV-X
 echo "Building TUV-x"
-
-# get & build the source code of TUV-x
-
-cd ${TUVX_HOME}
-git clone git@github.com:NCAR/tuv-x.git
```

Comment:
Isn't it still necessary to clone the TUV-x repo here?

Reply:
agreed

Reply:
I added the download script and pushed the changes. Apologies for the oversight

Reply:
no problem, looks great!

Reply:
@AdityaDendukuri @mattldawson @K20shores Is it just my problem that I still see the `git clone` command in both shell scripts?

Reply:
It should be in both scripts. The problem was that it was missing in one of them. Is that what you were asking?

Reply:
So if a user just download this script, then we need to `git clone` the TUV-x repo; if a user already clones the TUV-x repo, we could just set `TUVX_HOME` to the cloned repo and avoid the `git clone` here. Is that right?

Reply:
We could do that if you want, we would just need to update the `README` with those instructions. I think it was this way because you can also just download the script right from GitHub without cloning. But, I'm fine with either approach.

Reply:
Ah, that makes sense. I am fine with the current method but just want to make sure that I understand its usage correctly. Thanks for the clarification.